### PR TITLE
doc/md: add guide for typeorm schema loader

### PR DESCRIPTION
**Pull request by ronenlu**

### CODE REVIEW

File: doc/md/guides/orms/typeorm.mdx [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+the [migrations](https://typeorm.io/migrations)
+methodology, which is a more robust way to manage your database schema.
+TypeORM allows users to [generate migration](https://typeorm.io/migrations#generating-migrations)
+files based on the difference between the current state of the database and the desired state.
+
+In order to apply the schema migrations to the CI/CD pipeline, developers need to
+manually craft scripts that will execute the migrations against the production database and handle failures manually.
+
+Atlas has many features that can help developers manage their database
+[CI/CD pipelines](http://localhost:3000/guides/modern-database-ci-cd) more easily:
```
Comment (rotemtam) [inline doc/md/guides/orms/typeorm.mdx:33]: link (localhost)

File: doc/md/guides/orms/typeorm.mdx [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+
+* CI for schema changes - Atlas can be used during CI to make sure you never merge a pull request
```
Comment (rotemtam) [inline doc/md/guides/orms/typeorm.mdx:35]: in this section add links to relevant pages in the docs

Comment (rotemtam) [inline doc/md/guides/orms/typeorm.mdx:35]:
```suggestion
* Declarative migrations - use a Terraform-like `atlas schema apply` flow to manage your database.
* Diff policies - provide Atlas with policies for planning schema changes (such as "always create index concurrently" or "do not drop columns")
* CI for schema changes - Atlas can be used during CI to make sure you never merge a pull request
```

File: doc/md/guides/orms/typeorm.mdx [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+```
+
+### Standalone vs Script mode
+
+The Atlas TypeORM Provider can be used in two modes:
+* **Standalone** - If all of your TypeORM entities exist in a single Node module,
+you can use the provider directly to load your TypeORM schema into Atlas.
+* **Script** - In other cases, you can use the provider as an npm package to write a script
+ that loads your TypeORM schema into Atlas.
+### Standalone mode
```
Comment (rotemtam) [inline doc/md/guides/orms/typeorm.mdx:87]:
```suggestion

### Standalone mode
```

File: doc/md/guides/orms/typeorm.mdx [hunk 4]
```diff
@@ -0,0 +1,428 @@
...
+## TL;DR
+* [TypeORM](https://typeorm.io/) is an ORM for TypeScript and JavaScript that can run in many platforms
+  (Node.js, Browser, React Native, NativeScript, and more).
+* [Atlas](https://atlasgo.io) is an open-source tool for inspecting, planning, linting and
+  executing schema changes to your database.
+* Developers using TypeORM can use Atlas to automatically plan schema migrations
+  for them, based on the desired state of their schema instead of crafting them by hand.
+
+## Automatic migration planning for TypeORM
+TypeORM is a popular ORM widely used in the Node.js community. TypeORM allows users to
```
Comment (rotemtam) [inline doc/md/guides/orms/typeorm.mdx:19]:
```suggestion

TypeORM is a popular ORM widely used in the Node.js community. TypeORM allows users to
```

File: doc/md/guides/orms/typeorm.mdx [hunk 5]
```diff
@@ -0,0 +1,428 @@
...
+## Usage
+
+Atlas supports a [versioned migrations](/concepts/declarative-vs-versioned#versioned-migrations)
+workflow, where each change to the database is versioned and recorded in a migration file. You can use the
+`atlas migrate diff` command to automatically generate a migration file that will migrate the database
+from its latest revision to the current TypeORM schema.
+
+Suppose we have the following TypeORM `entities` directory, with two entities `Blog` and `User`:
+
+<Tabs>
```
Comment (rotemtam) [inline doc/md/guides/orms/typeorm.mdx:221]: something seems duplicate here?

**Review by rotemtam — COMMENTED** [review]
Looks good, see minor comments

File: doc/md/guides/orms/typeorm.mdx (hunk 5 above)
Comment (ronenlu) [reply doc/md/guides/orms/typeorm.mdx:221]: it because Im having tab inside tab, its ok.

**Review by rotemtam — APPROVED** [review]

**Review by rotemtam — APPROVED** [review]

**Pull request merged**